enhancements/ingress/admin-configured-ca-bundle.md

```diff
@@ -0,0 +1,242 @@
+---
+title: admin-configured-ca-bundle
+authors:
+  - "@bhb"
+reviewers:
+  - "@Miciah" ## reviewer for cluster-ingress-operator component
+  - "@tgeer" ## reviewer for CFE
+approvers:
+  - "@jerpeter1" ## approver for CFE
+  - "@Miciah" ## approver for cluster-ingress-operator component
+api-approvers:
+  - None
+creation-date: 2023-11-02
+last-updated: 2023-11-02
+tracking-link:
+  - https://issues.redhat.com/browse/RFE-2182
+  - https://issues.redhat.com/browse/OCPSTRAT-431
+  - https://issues.redhat.com/browse/NE-761
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Support for admin configured CA trust bundle in Ingress Operator
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of OpenShift
+to configure trusted Certificate Authorities(CA) to verify the service's 
+certificate by OpenShift Router for the re-encrypted traffic.
+
+## Motivation
+
+Openshift administrator when creating a route for a service with re-encrypt termination
+type requires to provide CA certificate when a service is configured to use certificate
+signed by a CA different from the OpenShift CA to verify the service's certificate.
+And when the same CA is used by the administrator for signing several services'
+certificates it is unnecessary to configure the same CA certificate content as destinationCA
+in every created route object. And is also tiresome task having to rotate the
+certificate when expired, compromised or for any other reason in each route object. Instead,
+if an administrator could define a ConfigMap with the list of CA certificates
+and OpenShift Ingress Controller used it along with OpenShift CA to create a common 
+CA bundle for verifying service's certificates it would make managing routes a bit simpler.
+
+### User Stories
+- As an OpenShift cluster administrator, I want to configure admin or the enterprise
```

Comment:
What do you mean by "admin or the enterprise"?  Are these two things the same or different?

Reply:
Both mean the same, thought to use both the terms as put in the SDFC description. I will update it to just `admin` since it's the term used in the proposal to refer user provided CA certificate bundle.